tests/e2e/specs/wcpay/merchant/merchant-disputes-submit-losing.spec.js

```diff
@@ -40,46 +40,44 @@ describe.skip( 'Disputes > Submit losing dispute', () => {
 	} );
 
 	it( 'should process and confirm a losing dispute', async () => {
-		// Pull out and follow the link to avoid working in multiple tabs
-		const paymentDetailsLink = await page.$eval(
-			'p.order_number > a',
-			( anchor ) => anchor.getAttribute( 'href' )
-		);
-
-		await merchantWCP.openPaymentDetails( paymentDetailsLink );
+		// Click the order dispute notice.
+		await expect( page ).toClick( '[type="button"]', {
+			text: 'Respond now',
+		} );
+		await page.waitForNavigation( {
+			waitUntil: 'networkidle0',
+		} );
 
-		// Verify we have a dispute for this purchase
-		await expect( page ).toMatchElement( 'li.woocommerce-timeline-item', {
-			text: 'Payment disputed as Product not received.',
+		// Verify we see the dispute details on the transaction details page.
+		await expect( page ).toMatchElement( '.dispute-notice', {
+			text: 'The cardholder claims the product was not received',
 		} );
 
-		// Accept the dispute
-		await merchantWCP.openAcceptDispute();
+		// Open the accept dispute modal.
+		await evalAndClick( '[data-testid="open-accept-dispute-modal-button"' );
+		await uiLoaded();
+		// Click the accept dispute button.
+		await evalAndClick( '[data-testid="accept-dispute-button"]' );
+		// Wait for the accept POST request to resolve and the status chip to update with the new status.
+		await expect( page ).toMatchElement( '.chip', {
+			text: 'Disputed: Lost',
+			timeout: 10000,
+		} );
 
-		// If webhooks are not received, the dispute status won't be updated in the dispute list page resulting in test failure.
-		// Workaround - Open payment details page again and check dispute's status.
-		await merchantWCP.openPaymentDetails( paymentDetailsLink );
+		// Check the dispute details footer
+		await expect( page ).toMatchElement(
+			'.transaction-details-dispute-footer *',
+			{
+				text: 'This dispute was accepted and lost',
+			}
+		);
 
 		// Confirm buttons are not present anymore since a dispute has been accepted.
 		await expect( page ).not.toMatchElement(
-			// eslint-disable-next-line max-len
-			'div.transaction-details-dispute-details-body div.transaction-details-dispute-details-body__actions button.components-button.is-primary',
-			{
-				text: 'Challenge dispute',
-			}
+			'[data-testid="challenge-dispute-button"]'
```

Comment:
`data-testid` is a common pattern from [react-testing-library](https://testing-library.com/docs/queries/bytestid), although it is the [_least_ preferred method of querying an element](https://testing-library.com/docs/queries/about#priority) (user-facing methods are preferred).

> In the spirit of [the guiding principles](https://testing-library.com/docs/guiding-principles), it is recommended to use this (`data-testid`) only after the other queries don't work for your use case.
> ...
> That said, they are way better than querying based on DOM structure or styling css class names.

The WooCommerce guide [Best Practices for Writing and Running End-to-End Tests](https://developer.woocommerce.com/testing-extensions-and-maintaining-quality-code/best-practices-for-writing-and-running-end-to-end-tests/#include-your-own-automation-ids) recommends a similar approach:

> Include your own automation IDs
> To avoid maintaining selectors in the future, it’s always a good idea to include your own data selector, such as `test-automation-id`
> ...
> instead of generally searching for visible elements on the page and using their IDs, classes, and XPaths.